### ISSUE TITLE
smoothie.d.ts missing elements

### ISSUE THREAD

**Issue by oosmos** [joewalnes/smoothie#75]
The `TimeSeries` class is missing public members:
```javascript
minValue: number;
maxValue: number;
```
The `SmoothieChart` class is missing public member:
```javascript
options: IChartOptions;
```
Use Case:
```javascript
  yRangeFunction: (): Smoothie.IRange => {
    const Min = Math.min(line1.minValue, line2.minValue) - 5;  // Access minValue
    const Max = Math.max(line1.maxValue, line2.maxValue) + 5;  // Access maxValue
    smoothie.options.grid.verticalSections = Max - Min;        // Change element within options

    return {max: Max, min: Min};
  },
```

**Comment by drewnoakes** [comment]
Can you make a PR?

**Comment by drewnoakes** [comment]
Closed by #76.
